src/styles.css

```diff
@@ -2,16 +2,32 @@
   display: none;
 }
 @media print {
+  html, body {
+    width:210mm;
+  }
+  .printRender{
+    width: 100%;
+    background-color: #fff;
+    z-index: 10000000;
+    height: 100vh;
+    position: absolute;
+    top: 0;
+    left: 0
+  }
+  .hiddenAll{
+    visibility: hidden;
+  }
+  .hiddenAll .print {
+    visibility: hidden;
+  }
   .wrap._loose:not(._invert) {
     visibility: hidden;
   }
   .print {
     visibility: visible; /* visible element even nested in the hidden one */
   }
   .print._main {
-    position: fixed;
-    left: 0;
-    top: 0;
+    visibility: visible !important;
```

Comment:
why you're using important?

visibility: visible can override parent's visibility: hidden out of the box w/o important hacks

Reply:
Yes, we don't need important here